packages/expo-updates/ios/EXUpdates/Database/EXUpdatesDatabase.m

```diff
@@ -56,17 +56,18 @@ - (void)dealloc
 
 - (void)addUpdate:(EXUpdatesUpdate *)update error:(NSError ** _Nullable)error
 {
-  NSString * const sql = @"INSERT INTO \"updates\" (\"id\", \"scope_key\", \"commit_time\", \"runtime_version\", \"metadata\", \"status\" , \"keep\")\
-  VALUES (?1, ?2, ?3, ?4, ?5, ?6, 1);";
+  NSString * const sql = @"INSERT INTO \"updates\" (\"id\", \"scope_key\", \"commit_time\", \"runtime_version\", \"manifest\", \"status\" , \"keep\", \"last_accessed\")\
+  VALUES (?1, ?2, ?3, ?4, ?5, ?6, 1, ?7);";
 
   [self _executeSql:sql
            withArgs:@[
                       update.updateId,
                       update.scopeKey,
-                      @([update.commitTime timeIntervalSince1970] * 1000),
+                      update.commitTime,
```

Comment:
Why are we dropping the conversion to INT? Is this autodetected by sqlite?

Reply:
A database utility function converts NSDates to UNIX timestamps measured in milliseconds.